services/galley/src/Galley/Cassandra/Queries.hs

```diff
@@ -212,28 +213,28 @@ updateTeamSplashScreen = {- `IF EXISTS`, but that requires benchmarking -} "upda
 
 -- Conversations ------------------------------------------------------------
 
-selectConv ::
-  PrepQuery
-    R
-    (Identity ConvId)
-    ( ConvType,
-      Maybe UserId,
-      Maybe (C.Set Access),
-      Maybe AccessRoleLegacy,
-      Maybe (C.Set AccessRole),
-      Maybe Text,
-      Maybe TeamId,
-      Maybe Bool,
-      Maybe Milliseconds,
-      Maybe ReceiptMode,
-      Maybe ProtocolTag,
-      Maybe GroupId,
-      Maybe Epoch,
-      Maybe (Writetime Epoch),
-      Maybe CipherSuiteTag,
-      Maybe GroupConvType
-    )
-selectConv = "select type, creator, access, access_role, access_roles_v2, name, team, deleted, message_timer, receipt_mode, protocol, group_id, epoch, WRITETIME(epoch), cipher_suite, group_conv_type from conversation where conv = ?"
+type ConvRow =
```

Comment:
would be nice to use https://hackage.haskell.org/package/cql-4.0.4/docs/Database-CQL-Protocol.html#v:recordInstance
but maybe too much for this PR

Reply:
Yeah, let's do it in a separate PR.